python/ray/_private/test_utils.py

```diff
@@ -1810,3 +1810,5 @@ def safe_write_to_results_json(
     with open(test_output_json_tmp, "wt") as f:
         json.dump(result, f)
     os.replace(test_output_json_tmp, test_output_json)
+    logger.info(f"Wrote results to {test_output_json}")
```

Comment:
This probably doesn't print anything in tests actually cuz the default logger level is warning in Python. Maybe consider to just use print?